.changeset/eleven-keys-work.md

```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': patch
+'@swisspost/design-system-styles': patch
+---
+
+Updated the style of Headings (h1-h6) using design tokens.
```

Comment:
I suggest not to mention the tokens in the change log, as they are part of our internal technology that users don't necessarily need to be aware of:
```suggestion
Updated the style of headings (h1-h6).
```